docs/reference/ml/anomaly-detection/apis/put-job.asciidoc

```diff
@@ -63,6 +63,10 @@ include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=analysis-config]
 `bucket_span`:::
 (<<time-units,time units>>)
 include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=bucket-span]
++
+deprecated:[8.1, "Setting `bucket_span` to a value that is neither an integral multiple
+nor divisor of the number of seconds in a day is deprecated and will be disallowed in a
+future major version."]
```

Comment:
This might be clearer to the average person if there was less mathematical terminology, for example:

```suggestion
deprecated:[8.1, "Setting `bucket_span` such that it is neither a whole number of days
nor that there are a whole number of buckets in one day is deprecated and will be
disallowed in a future version."]
```

Do you have an opinion @lcawl?

Reply:
I agree that this suggestion is easier to understand. I also think this detail is important enough to include in the body of the description, rather than in a deprecation popover.  I'll add a commit for consideration.